src/DssSpell.sol

```diff
@@ -55,172 +36,62 @@ contract DssSpellAction is DssAction {
-
-        // give the urn permissions on the join adapter
-        DssExecLib.authorize(MCD_JOIN_RWA001_A, RWA001_A_URN);
-
-        // set up the urn
-        Hopeable(RWA001_A_URN).hope(RWA001_OPERATOR);
-
-        // set up output conduit
-        Hopeable(RWA001_A_OUTPUT_CONDUIT).hope(RWA001_OPERATOR);
-
-        // Authorize the SC Domain team deployer address on the output conduit during introductory phase.
-        //  This allows the SC team to assist in the testing of a complete circuit.
-        //  Once a broker dealer arrangement is established the deployer address should be `deny`ed on the conduit.
-        Kissable(RWA001_A_OUTPUT_CONDUIT).kiss(SC_DOMAIN_DEPLOYER_07);
-
-        // add RWA-001 contract to the changelog
-        DssExecLib.setChangelogAddress("RWA001", RWA001_GEM);
-        DssExecLib.setChangelogAddress("PIP_RWA001", pip);
-        DssExecLib.setChangelogAddress("MCD_JOIN_RWA001_A", MCD_JOIN_RWA001_A);
-        DssExecLib.setChangelogAddress("MIP21_LIQUIDATION_ORACLE", MIP21_LIQUIDATION_ORACLE);
-        DssExecLib.setChangelogAddress("RWA001_A_URN", RWA001_A_URN);
-        DssExecLib.setChangelogAddress("RWA001_A_INPUT_CONDUIT", RWA001_A_INPUT_CONDUIT);
-        DssExecLib.setChangelogAddress("RWA001_A_OUTPUT_CONDUIT", RWA001_A_OUTPUT_CONDUIT);
+        // Rates Proposal
+        DssExecLib.setIlkStabilityFee("UNIV2DAIETH-A", THREE_PCT, true);
+        DssExecLib.setIlkStabilityFee("UNIV2USDCETH-A", THREE_PT_FIVE_PCT, true);
+        DssExecLib.setIlkStabilityFee("UNIV2WBTCETH-A", FOUR_PT_FIVE_PCT, true);
+        DssExecLib.setIlkStabilityFee("UNIV2ETHUSDT-A", FIVE_PCT, true);
+        DssExecLib.setIlkStabilityFee("UNIV2LINKETH-A", FIVE_PCT, true);
+        DssExecLib.setIlkStabilityFee("UNIV2UNIETH-A", FIVE_PCT, true);
+        DssExecLib.setIlkStabilityFee("UNIV2AAVEETH-A", FIVE_PCT, true);
+        DssExecLib.setIlkStabilityFee("MANA-A", FOUR_PCT, true);
+        DssExecLib.setIlkStabilityFee("UNIV2WBTCDAI-A", TWO_PCT, true);
+        DssExecLib.setIlkStabilityFee("RENBTC-A", FIVE_PCT, true);
+
+        // Onboarding ETH-C
+        CollateralOpts memory ETH_C = CollateralOpts({
+            ilk: "ETH-C",
+            gem: DssExecLib.getChangelogAddress("ETH"),
+            join: MCD_JOIN_ETH_C,
+            flip: MCD_FLIP_ETH_C,
+            pip: DssExecLib.getChangelogAddress("PIP_ETH"),
+            isLiquidatable: true,
+            isOSM: true,
+            whitelistOSM: false,
+            ilkDebtCeiling: 100 * MILLION,
+            minVaultAmount: 2000,
+            maxLiquidationAmount: 50000,
+            liquidationPenalty: 1300,
+            ilkStabilityFee: THREE_PT_FIVE_PCT,
+            bidIncrease: 300,
+            bidDuration: 4 hours,
+            auctionDuration: 4 hours,
+            liquidationRatio: 17500
+        });
+        addNewCollateral(ETH_C);
+        DssExecLib.setIlkAutoLineParameters("ETH-C", 2000 * MILLION, 100 * MILLION, 12 hours);
```

Comment:
This can't be right.  This `gap` is way too large.  I don't think we should do it this way.  If we want to grow this collateral type, especially where we have no keepers yet, we still need to grow up to it.  We are not using this module correctly.  We want the `gap` to limit our bad debt exposure in a sharp pullback.  I know the CR is higher, and we want people to migrate, but 100 million is too high IMHO.  I would suggest something at 10 million or less, and only something more aggressive (20 million) for a short time while we migrate.  FWIW, we are doing this wrong on `ETH-A` too.